kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/ClusterCondition.java

```diff
@@ -31,4 +38,16 @@ static ClusterCondition filterKindNotKnown(String cluster, String filterName, St
                 String.format("Filter \"%s\" has a kind %s that is not known to this operator.", filterName, kind));
     }
 
+    static ClusterCondition targetClusterRefNotExists(String cluster, TargetCluster targetCluster) {
+        return new ClusterCondition(cluster, ConditionType.Accepted, Status.FALSE, INVALID,
+                String.format("Target Cluster \"%s\" does not exist.", kubeName(targetCluster).orElse("<unknown>")));
+    }
+
+    @NonNull
+    private static Optional<String> kubeName(TargetCluster targetCluster) {
+        return Optional.ofNullable(targetCluster.getClusterRef())
+                .map(r -> "%s%s/%s".formatted(r.getKind().toLowerCase(Locale.ROOT), Optional.ofNullable(r.getGroup()).map(
+                        ".%s"::formatted).orElse(""), r.getName()));
```

Comment:
This is difficult to read. Can we do improve it so that the nesting is more obvious?